.github/workflows/stale-tasks.yml

```diff
@@ -0,0 +1,26 @@
+# This is a workflow that runs periodically to mark issues/prs with no recent activity as stale, it also closes/removes label based on following activity
+
+name: Mark and close stale issues/PRs
+on:
+  schedule:
+  - cron: '0 0 * * *' # runs daily at 12:00 am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-label: stale
+          stale-pr-label: stale
+          days-before-stale: 45
+          days-before-close: 45
+          remove-stale-when-updated: true
+          stale-issue-message: > 
+            Issue has been automatically marked as stale because it has been open for 45 days with no activity. Issue will be closed if no further activity occurs.
+            Thank you for your contributions.
```

Comment:
Is the Thank you for your contributions. cold? Should we add a line explaining why we use an automated action or perhaps a link to a page with explanation?

Reply:
I dont think we need to be too verbose, since this is fairly standard.

Reply:
I saw that in some github repos and hence the proposal. I can remove it if majority don't like it.

Reply:
I would not remove it, just wanted more warmer tone. If this matches what other folks are doing, fine as is.